app/views/teachers/index.html.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
add a title that says teachers or something like that just to know that we been redirected here and to not send an empty partial

Reply:
actually, this view is not necessary at all, I will delete it